src/Slider.js

```diff
@@ -304,16 +312,16 @@ export default class Slider extends PureComponent {
 
   _handleStartShouldSetPanResponder = (e: Object, /*gestureState: Object*/): boolean => {
     // Should we become active when the user presses down on the thumb?
-    return this._thumbHitTest(e);
+    return this.props.trackClickable ? true : this._thumbHitTest(e);
   };
 
   _handleMoveShouldSetPanResponder(/*e: Object, gestureState: Object*/): boolean {
     // Should we become active when the user moves a touch over the thumb?
     return false;
   };
 
-  _handlePanResponderGrant = (/*e: Object, gestureState: Object*/) => {
-    this._previousLeft = this._getThumbLeft(this._getCurrentValue());
+  _handlePanResponderGrant = (e: Object, gestureState: Object) => {
+    this._previousLeft = this.props.trackClickable ? gestureState.x0 - (this.state.thumbSize.width/2) : this._getThumbLeft(this._getCurrentValue());
```

Comment:
`gestureState.x0` is relative to the root component, this should be `e.nativeEvent.locationX` which is relative to the panResponder component.

Reply:
also need to change:
this.state.thumbSize -> this.props.thumbTouchSize

Reply:
@dummerbd @sreym Thanks for the review. I will make the changes and test them once. Haven't been working on RN lately.